tools/tscdocgen/main.go

```diff
@@ -487,27 +487,41 @@ func (e *emitter) emitMarkdownModule(name string, mod *module, root bool) error
 		}
 	}
 
+	hasModules := len(modules) > 0
+	hasMembers := len(members) > 0
+	hasNamespaces := len(namespaces) > 0
+	hasResources := len(resources) > 0
+	hasDataSources := len(dataSources) > 0
+
+	metaDescription := "Explore "
+	if root {
+		metaDescription += "members of the " + linktitle + " package."
+	} else {
+		metaDescription += "members of the " + linktitle + " module in the " + e.pkgname + " package."
```

Comment:
Should this be `e.pkg` instead of `e.pkgname` so it shows up as `@pulumi/packet` instead of `packet`?

Reply:
Sure thing. I'll change it.